src/test/java/integration/DiscountPerCounterpartyControllerIT.java

```diff
@@ -165,13 +162,13 @@ public void updateDiscountPerCounterparty() throws Exception {
         then().
                 body("discountUuid", equalTo(newDiscount.getUuid().toString())).
                 statusCode(SC_OK);
-        long timeFinished = currentTimeMillis();
+        LocalDateTime timeFinished = now();
 
         DiscountPerCounterparty updatedDiscountPerCounterparty = discountPerCounterpartyService
                 .getEntityByUuid(discountPerCounterparty.getUuid(), user);
-        long timeModified = updatedDiscountPerCounterparty.getLastModified().getTime();
+        LocalDateTime timeModified = updatedDiscountPerCounterparty.getLastModified();
 
-        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished >= timeModified && timeModified >= timeStarted);
+        assertTrue(WRONG_LAST_MODIFIED_MESSAGE, timeFinished.isAfter(timeModified) && timeModified.isAfter(timeStarted));
```

Comment:
Plz, change. isAfter is not enough. It should do ">=".
Plz, create helper method assertDateBetween(date, dateFrom, dateTo) in the new class AssertHelper(package util).